src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java

```diff
@@ -143,7 +145,7 @@ public void archive(FilePath workspace, Launcher launcher, BuildListener listene
                 String blobPath = getBlobPath(path);
                 Blob blob = blobStore.blobBuilder(blobPath).build();
                 blob.getMetadata().setContainer(provider.getContainer());
-                blob.getMetadata().getContentMetadata().setContentType(contentTypes.get(entry.getKey()));
+                blob.getMetadata().getContentMetadata().setContentType(contentTypes.get(entry.getValue()));
```

Comment:
@seansaleh this was the only merge conflict I needed to resolve. You can see in line 129/131 above the `contentTypes` map was changed to using `artifacts.values()` as the key(s), so I updated this line to pull the proper values